src/app.ts

```diff
@@ -33,3 +33,17 @@ export interface SelfIdentity {
   selfId: string
   selfContext: string
 }
+
```

Comment:
can’t we use the server api types?

Reply:
Do you mean for feature id and type?

Reply:
Removed instances of featureId and featureType.